lib/shark_client.js

```diff
@@ -129,6 +127,21 @@ function _request(opts, cb) {
     }
 }
 
+/*
+ * Use information about the object to craft the storage path.
+ */
+function storagePath(opts) {
+    var storageLayoutVersion = '/v' + opts.storageLayoutVersion + '/';
```

Comment:
How about having this throw an exception if storageLayoutVersion is not a known version?